### DISCUSSION TITLE
How do I configure exports to use module from package.json before exports when compiling files

### DISCUSSION THREAD

**Discussion by gyf30827** [webpack/webpack#16260] [Q&A]
# Bug report

**What is the current behavior?**
webpack config
![image](https://user-images.githubusercontent.com/22958382/190596857-b15e5db3-40ba-456f-b4cc-c1ddd360f109.png)
But export is introduced at compile time
![image](https://user-images.githubusercontent.com/22958382/190597021-e8ad68b1-777b-4269-a2ab-4306e9e66215.png)

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

module introduced in package.jsong have higher priority than exports

**Other relevant information:**
webpack version:  
5.74.0
Node.js version: 
v16.16.0
Operating System: 
macos
Additional tools:

**Comment by alexander-akait** [comment]
Please fill out `If the current behavior is a bug, please provide the steps to reproduce.`
